code/frontend/src/pages/chat/Chat.tsx

```diff
@@ -168,12 +167,12 @@ const Chat = () => {
           "Azure Speech subscription key or region is not defined."
         );
       } else {
-        const speechConfig = SpeechConfig.fromSubscription(
-          subscriptionKey,
-          serviceRegion
-        );
-        const audioConfig = AudioConfig.fromDefaultMicrophoneInput();
-        const recognizer = new SpeechRecognizer(speechConfig, audioConfig);
+        const recognizer = multiLingualSpeechRecognizer(subscriptionKey, serviceRegion, [
+          "en-US",
+          "fr-FR",
+          "de-DE",
+          "it-IT"
+        ]);
```

Comment:
Also, as mentioned on the PR description, the plan is to move this hardocded list of languages into the deployment config under #317 .